mu/harness/pm.py

```diff
@@ -123,6 +147,31 @@ def log_all(self):
                 continue
         self.logger.unsubscribe(sub)
 
+    def sim_list(self):
+        ret = []
+        for fd in self.fd_to_sim:
+            ret.append({
+                'fd': fd,
+                'sim': self.fd_to_sim[fd].__class__.__name__,
+            })
+        return ret
+
+    def sim_cat(self):
+        sim_files = os.listdir(os.path.join(REPO_DIR, 'mu/sims'))
+        catalog = {}
+        for sim_file in sim_files:
+            if not sim_file.endswith('.py'):
+                continue
+            sim_name = sim_file[:len(sim_file)-3]
+            mod_name = 'mu.sims.{}'.format(sim_name)
+            importlib.import_module(mod_name)
+            sim_classes = inspect.getmembers(sys.modules[mod_name], inspect.isclass)
+            for sim_class in sim_classes:
+                if SubSim in sim_class[1].__bases__ or sim_class[0] in catalog:
```

Comment:
Might want to check if `BoardSim` is in the class hierarchy rather than just that `SubSim` isn't: `BoardSim in inspect.getmro(sim_class[1])`. `getmro` gets all the ancestor classes (in **m**ethod-**r**esolution **o**rder) rather than just 1 level up like `__bases__`.